src/DtoGenerator/DtoGenerator.Logic/UI/PropertySelectorViewModel.cs

```diff
@@ -31,66 +41,152 @@ public class PropertySelectorViewModel : ViewModelBase
 
             instance.AddDataContract = await EntityParser.HasDataContract(existingDto);
             instance.AddDataAnnotations = await EntityParser.HasDataAnnotations(existingDto);
+            if (instance.RelatedEntiesByObject)
+                instance.MapEntitiesById = await EntityParser.HasMapEntitiesById(existingDto);
 
             return instance;
         }
-
+        private Document _doc;
         private PropertySelectorViewModel()
         {
             this.GenerateMapper = true;
         }
 
-        private bool _generateMapper;
+        public class GeneratorProperties
+        {
+            public bool generateMapper;
```

Comment:
I would like to see those properties as auto properties, with capital letter - public bool GenerateMapper {get;set;}